setup.py

```diff
@@ -28,7 +28,7 @@
     exec(f.read(), metadata)
 
 tests_require = ['attrs==20.3.0', 'pytest==6.2.5', 'pytest-xdist==2.2.1', 'pytest-forked==1.3.0',
-                 'python3-indy==1.16.0.post236', 'pytest-asyncio==0.14.0']
+                'pytest-asyncio==0.14.0', 'indy_vdr==0.4.0.dev5', 'aries-askar==0.2.7', 'indy-credx==0.3.1']
```

Comment:
Latest releases are:
indy-vdr: [0.4.3](https://github.com/hyperledger/indy-vdr/releases/tag/v0.4.3)
askar: [0.4.3](https://github.com/openwallet-foundation/askar/releases/tag/v0.4.3)
indy-credx: [1.1.1](https://github.com/hyperledger/indy-shared-rs/releases/tag/v1.1.1)

Reply:
Looks like `0.4.3` did not get published successfully:
- https://github.com/hyperledger/indy-vdr/releases/tag/v0.4.3
- https://pypi.org/project/indy-vdr/#history

Reply:
Use `0.4.2` for now.

Reply:
Created an issue for the publishing issue here; https://github.com/hyperledger/indy-vdr/issues/335